service/raft/group0_state_machine.cc

```diff
@@ -140,6 +140,28 @@ static future<> write_mutations_to_database(storage_proxy& proxy, gms::inet_addr
     }
 }
 
+group0_state_machine::modules_to_reload group0_state_machine::find_modules_to_reload(const std::vector<canonical_mutation>& mutations) {
+    modules_to_reload modules;
+
+    for (auto& mut: mutations) {
+        if (mut.column_family_id() == db::system_keyspace::service_levels_v2()->id()) {
+            modules.insert(reload_module::service_level_controller);
+        }
+    }
+
+    return modules;
+}
+
+future<> group0_state_machine::reload_modules(modules_to_reload modules) {
+    for (auto& module: modules) {
+        switch (module) {
+            case reload_module::service_level_controller:
```

Comment:
I imagine that the function should be organized as a sequence of ifs, checking for whether it's necessary to reload some module, and reload it if the appropriate flag is set in the `modules` variable. This will allow us to control the order of reloading things which will be useful later.